app/models/autoform_registration.rb

```diff
@@ -0,0 +1,14 @@
+class AutoformRegistration < Registration
+  EXTRA_FIELDS = { domain: 'entry.591019594' }.freeze
+
+  attr_accessor :domain
+  validates :domain, presence: true, on: :submit
```

Comment:
Tento `on: :submit` context je tam na co? Ved to chceme asi vzdy nie?

Reply:
good point. par commitov dozadu tam bolo este aj `on: :render`, a ked som to vyhodil tak som na toto zabudol.